src/.vuepress/locales/details/locale_detail_ja.js

```diff
@@ -1,7 +1,7 @@
 const sidebar_ja = require("./sidebar/sidebar_ja");
 const nav_ja = require("./nav/nav_ja");
 
-let ja_locales = {
+module.exports = ja_locales = {
```

Comment:
This is not valid JS. I think you meant to remove `ja_locales = `

Reply:
Check please